electrumsv/wallet.py

```diff
@@ -328,6 +272,9 @@ def is_synchronized(self):
         return (self._synchronized_event.is_set() and
                 not (self.network and self.missing_transactions()))
 
+    def load_utxo_cache(self):
+        self._datastore.utxos.get_spendable_coins_cached()
```

Comment:
We use get for when a value is returned. I would prefer "cache spendable coins" or "populate_spendable_coin_cache".

Reply:
Removed - not even used by anything